### ISSUE TITLE
Remove body from Discord notification & other

### ISSUE THREAD

**Issue by damien67** [crazy-max/diun#468]
Hi, I am trying to switch my notifications from Gotify to Discord. 

On Gotify, they are simple and concise, such as Image update for docker.io/crazymax/diun:edge as title, and a short "Docker tag docker.io/crazymax/diun:edge which you subscribed to ..... has been updated".

On Discord, I get the whole thing as shown here https://crazymax.dev/diun/assets/notif/discord-2.png.
I am not so interested in the body part...I would rather enhance the title with the created date, but the box takes too much space and having many container, it is just hard to follow. 

Is it possible?

Bonus questions...

```
Fri, 20 Aug 2021 16:09:56 EDT INF New image found image=docker.io/crazymax/diun:edge provider=docker
Fri, 20 Aug 2021 16:09:56 EDT ERR Discord notification failed error="unexpected HTTP status 429: &{%!s(func()=0x62b100) {%!s(*http.http2clientStream=&{0xc000103b00 0xc00003ce00 <nil> 15 0xc000744240 {{0 0} {{} 0xc0006ea9c8 {0 0 0 <nil> <nil>} 824640973312} 0xc0005dc1c0 0 0xc000072070 <nil> <nil> 0x80fb00} true true 0x80fa00 {[] 64563 0xc000103b60} {[] 4194213 0xc000103b70} 91 <nil> 0xc000072be0 false 0xc000660de0 <nil> 0xc000660e40 true true false 0 map[] 0xc000490078})} %!s(func() bool=0x807260)}" image=docker.io/crazymax/diun:4.7.0
Fri, 20 Aug 2021 16:10:27 EDT INF New image found image=docker.io/crazymax/diun:4.5.0 provider=docker
Fri, 20 Aug 2021 16:10:27 EDT INF New image found image=docker.io/crazymax/diun:4.4.0 provider=docker
Fri, 20 Aug 2021 16:10:27 EDT INF New image found image=docker.io/crazymax/diun:4.4.1 provider=docker
Fri, 20 Aug 2021 16:10:27 EDT INF New image found image=docker.io/crazymax/diun:4.6.0 provider=docker
Fri, 20 Aug 2021 16:10:42 EDT INF New image found image=docker.io/crazymax/diun:4.2.0 provider=docker
Fri, 20 Aug 2021 16:10:42 EDT INF New image found image=docker.io/crazymax/diun:4.3.1 provider=docker
Fri, 20 Aug 2021 16:10:42 EDT INF New image found image=docker.io/crazymax/diun:4.3.0 provider=docker
Fri, 20 Aug 2021 16:10:42 EDT INF New image found image=docker.io/crazymax/diun:4.18.0 provider=docker
Fri, 20 Aug 2021 16:10:42 EDT INF New image found image=docker.io/crazymax/diun:4.17.0 provider=docker
Fri, 20 Aug 2021 16:10:42 EDT INF New image found image=docker.io/crazymax/diun:4.19.0 provider=docker
Fri, 20 Aug 2021 16:10:42 EDT ERR Discord notification failed error="unexpected HTTP status 429: &{%!s(func()=0x62b100) {%!s(*http.http2clientStream=&{0xc000103b00 0xc000552600 <nil> 31 0xc00041b560 {{0 0} {{} 0xc000385788 {0 0 0 <nil> <nil>} 824637413312} 0xc000c8c880 0 0xc000072070 <nil> <nil> 0x80fb00} true true 0x80fa00 {[] 64563 0xc000103b60} {[] 4194213 0xc000103b70} 91 <nil> 0xc000072be0 false 0xc00065e600 <nil> 0xc00065e660 true true false 0 map[] 0xc0005a2c48})} %!s(func() bool=0x807260)}" image=docker.io/crazymax/diun:4.3.1
Fri, 20 Aug 2021 16:11:13 EDT INF New image found image=docker.io/crazymax/diun:4.15.2 provider=docker
Fri, 20 Aug 2021 16:11:13 EDT INF New image found image=docker.io/crazymax/diun:4.15.1 provider=docker
Fri, 20 Aug 2021 16:11:13 EDT INF New image found image=docker.io/crazymax/diun:4.16.0 provider=docker
Fri, 20 Aug 2021 16:11:13 EDT INF New image found image=docker.io/crazymax/diun:4.16.1 provider=docker
Fri, 20 Aug 2021 16:11:28 EDT INF New image found image=docker.io/crazymax/diun:4.14.0 provider=docker
Fri, 20 Aug 2021 16:11:28 EDT INF New image found image=docker.io/crazymax/diun:4.15.0 provider=docker
Fri, 20 Aug 2021 16:11:28 EDT INF New image found image=docker.io/crazymax/diun:4.12.0 provider=docker
Fri, 20 Aug 2021 16:11:28 EDT INF New image found image=docker.io/crazymax/diun:4.13.0 provider=docker
Fri, 20 Aug 2021 16:11:28 EDT INF New image found image=docker.io/crazymax/diun:4.10.0 provider=docker
Fri, 20 Aug 2021 16:11:28 EDT INF New image found image=docker.io/crazymax/diun:4.11.0 provider=docker
Fri, 20 Aug 2021 16:11:29 EDT ERR Discord notification failed error="unexpected HTTP status 429: &{%!s(func()=0x62b100) {%!s(*http.http2clientStream=&{0xc000103b00 0xc000812c00 <nil> 59 0xc0003c2600 {{0 0} {{} 0xc0006eb788 {0 0 0 <nil> <nil>} 824640976832} 0xc00012efc0 0 0xc000072070 <nil> <nil> 0x80fb00} true true 0x80fa00 {[] 64561 0xc000103b60} {[] 4194213 0xc000103b70} 91 <nil> 0xc000072be0 false 0xc00065e600 <nil> 0xc00065e660 true true false 0 map[] 0xc000490bb8})} %!s(func() bool=0x807260)}" image=docker.io/crazymax/diun:4.11.0
Fri, 20 Aug 2021 16:11:59 EDT INF New image found image=docker.io/crazymax/diun:4.0.0 provider=docker
Fri, 20 Aug 2021 16:11:59 EDT INF New image found image=docker.io/crazymax/diun:4.1.1 provider=docker
Fri, 20 Aug 2021 16:11:59 EDT INF New image found image=docker.io/crazymax/diun:4.0.0-beta.4 provider=docker
Fri, 20 Aug 2021 16:11:59 EDT INF New image found image=docker.io/crazymax/diun:4.1.0 provider=docker
Fri, 20 Aug 2021 16:12:15 EDT INF New image found image=docker.io/crazymax/diun:4.0.0-beta.3 provider=docker
Fri, 20 Aug 2021 16:12:15 EDT INF New image found image=docker.io/crazymax/diun:3.0.0-beta.4 provider=docker
Fri, 20 Aug 2021 16:12:15 EDT INF New image found image=docker.io/crazymax/diun:4.0.0-beta.1 provider=docker
Fri, 20 Aug 2021 16:12:15 EDT INF New image found image=docker.io/crazymax/diun:3.0.0-beta.5 provider=docker
Fri, 20 Aug 2021 16:12:15 EDT INF New image found image=docker.io/crazymax/diun:4.0.0-beta.2 provider=docker
Fri, 20 Aug 2021 16:12:15 EDT INF New image found image=docker.io/crazymax/diun:3.0.0 provider=docker
Fri, 20 Aug 2021 16:12:15 EDT ERR Discord notification failed error="unexpected HTTP status 429: &{%!s(func()=0x62b100) {%!s(*http.http2clientStream=&{0xc000103b00 0xc000404d00 <nil> 79 0xc000745020 {{0 0} {{} 0xc0006eb8e8 {0 0 0 <nil> <nil>} 824640977184} 0xc000abad40 0 0xc000072070 <nil> <nil> 0x80fb00} true true 0x80fa00 {[] 64563 0xc000103b60} {[] 4194213 0xc000103b70} 91 <nil> 0xc000072be0 false 0xc000c7cc00 <nil> 0xc000c7cc60 true true false 0 map[] 0xc0004dba58})} %!s(func() bool=0x807260)}" image=docker.io/crazymax/diun:3.0.0
Fri, 20 Aug 2021 16:12:46 EDT INF New image found image=docker.io/crazymax/diun:3.0.0-beta.1 provider=docker
Fri, 20 Aug 2021 16:12:46 EDT INF New image found image=docker.io/crazymax/diun:3.0.0-beta.2 provider=docker
Fri, 20 Aug 2021 16:12:46 EDT INF New image found image=docker.io/crazymax/diun:2.6.1 provider=docker
Fri, 20 Aug 2021 16:12:46 EDT INF New image found image=docker.io/crazymax/diun:3.0.0-beta.3 provider=docker
Fri, 20 Aug 2021 16:13:01 EDT INF New image found image=docker.io/crazymax/diun:2.3.0 provider=docker
Fri, 20 Aug 2021 16:13:01 EDT INF New image found image=docker.io/crazymax/diun:2.2.1 provider=docker
Fri, 20 Aug 2021 16:13:01 EDT INF New image found image=docker.io/crazymax/diun:2.2.0 provider=docker
Fri, 20 Aug 2021 16:13:01 EDT INF New image found image=docker.io/crazymax/diun:2.4.0 provider=docker
Fri, 20 Aug 2021 16:13:01 EDT INF New image found image=docker.io/crazymax/diun:2.5.0 provider=docker
Fri, 20 Aug 2021 16:13:01 EDT INF New image found image=docker.io/crazymax/diun:2.6.0 provider=docker
Fri, 20 Aug 2021 16:13:01 EDT ERR Discord notification failed error="unexpected HTTP status 429: &{%!s(func()=0x62b100) {%!s(*http.http2clientStream=&{0xc000103b00 0xc000813200 <nil> 99 0xc0009bd680 {{0 0} {{} 0xc0002acde8 {0 0 0 <nil> <nil>} 824636526112} 0xc0000a7280 0 0xc000072070 <nil> <nil> 0x80fb00} true true 0x80fa00 {[] 64563 0xc000103b60} {[] 4194213 0xc000103b70} 91 <nil> 0xc000072be0 false 0xc000897080 <nil> 0xc0008970e0 true true false 0 map[] 0xc0005507c8})} %!s(func() bool=0x807260)}" image=docker.io/crazymax/diun:2.6.0
Fri, 20 Aug 2021 16:13:32 EDT INF New image found image=docker.io/crazymax/diun:2.0.0 provider=docker
Fri, 20 Aug 2021 16:13:32 EDT INF New image found image=docker.io/crazymax/diun:1.4.0 provider=docker
Fri, 20 Aug 2021 16:13:32 EDT INF New image found image=docker.io/crazymax/diun:1.4.1 provider=docker
Fri, 20 Aug 2021 16:13:32 EDT INF New image found image=docker.io/crazymax/diun:2.1.0 provider=docker
Fri, 20 Aug 2021 16:13:42 EDT INF New image found image=docker.io/crazymax/diun:1.0.2 provider=docker
Fri, 20 Aug 2021 16:13:42 EDT INF New image found image=docker.io/crazymax/diun:1.0.1 provider=docker
Fri, 20 Aug 2021 16:13:42 EDT INF New image found image=docker.io/crazymax/diun:1.0.0 provider=docker
Fri, 20 Aug 2021 16:13:42 EDT INF New image found image=docker.io/crazymax/diun:1.3.0 provider=docker
Fri, 20 Aug 2021 16:13:42 EDT INF New image found image=docker.io/crazymax/diun:1.2.0 provider=docker
Fri, 20 Aug 2021 16:13:42 EDT INF New image found image=docker.io/crazymax/diun:1.1.0 provider=docker
Fri, 20 Aug 2021 16:13:42 EDT ERR Discord notification failed error="unexpected HTTP status 429: &{%!s(func()=0x62b100) {%!s(*http.http2clientStream=&{0xc000103b00 0xc000149b00 <nil> 113 0xc000a40d80 {{0 0} {{} 0xc000daf8e8 {0 0 0 <nil> <nil>} 824648071456} 0xc0004cc980 0 0xc000072070 <nil> <nil> 0x80fb00} true true 0x80fa00 {[] 64563 0xc000103b60} {[] 4194213 0xc000103b70} 91 <nil> 0xc000072be0 false 0xc00008b3e0 <nil> 0xc00008b440 true true false 0 map[] 0xc0005507c8})} %!s(func() bool=0x807260)}" image=docker.io/crazymax/diun:1.0.0
Fri, 20 Aug 2021 16:14:13 EDT INF New image found image=docker.io/crazymax/diun:0.4.0 provider=docker
Fri, 20 Aug 2021 16:14:13 EDT INF New image found image=docker.io/crazymax/diun:0.5.0 provider=docker
Fri, 20 Aug 2021 16:14:13 EDT INF New image found image=docker.io/crazymax/diun:0.3.2 provider=docker
Fri, 20 Aug 2021 16:14:13 EDT INF New image found image=docker.io/crazymax/diun:0.4.1 provider=docker
Fri, 20 Aug 2021 16:14:23 EDT INF New image found image=docker.io/crazymax/diun:0.1.0 provider=docker
Fri, 20 Aug 2021 16:14:23 EDT INF New image found image=docker.io/diameter/rtorrent-rutorrent:latest provider=docker
Fri, 20 Aug 2021 16:14:23 EDT INF New image found image=docker.io/crazymax/diun:0.1.1 provider=docker
Fri, 20 Aug 2021 16:14:23 EDT INF New image found image=docker.io/crazymax/diun:0.2.0 provider=docker
Fri, 20 Aug 2021 16:14:23 EDT INF New image found image=docker.io/crazymax/diun:0.3.0 provider=docker
Fri, 20 Aug 2021 16:14:23 EDT INF New image found image=docker.io/crazymax/diun:0.3.1 provider=docker
Fri, 20 Aug 2021 16:14:24 EDT ERR Discord notification failed error="unexpected HTTP status 429: &{%!s(func()=0x62b100) {%!s(*http.http2clientStream=&{0xc000103b00 0xc000404c00 <nil> 139 0xc000b06e40 {{0 0} {{} 0xc000385a48 {0 0 0 <nil> <nil>} 824637414016} 0xc000c8cac0 0 0xc000072070 <nil> <nil> 0x80fb00} true true 0x80fa00 {[] 64563 0xc000103b60} {[] 4194213 0xc000103b70} 91 <nil> 0xc000072be0 false 0xc000c7cd80 <nil> 0xc000c7cea0 true true false 0 map[] 0xc0005a2cd8})} %!s(func() bool=0x807260)}" image=docker.io/crazymax/diun:0.3.1
```

Why so many entries for Diun? I just need to know if there is a newer image than the one I am using... looks this this is the only container behaving this way. And what about unexpected HTTP status 429? API limit rate issue with Discord?

Thanks!!!

**Comment by crazy-max** [comment]
Please open one ticket per issue: https://github.com/crazy-max/diun/blob/master/.github/SUPPORT.md#writing-good-bug-reports-and-feature-requests

**Comment by damien67** [comment]
@crazy-max, thank you! I will report the Error 429 in another post.
